src/implementations/vanilla/components/tabs/tabs.js

```diff
@@ -334,6 +374,100 @@ export class Tabs {
     this.moreButton.setAttribute('aria-expanded', false);
     this.dropdown.classList.remove('ecl-tabs__dropdown--show');
   }
+
+  /**
+   * Tabs onKeydown handler.
+   * @param {Event} e
+   */
+  onKeydown(e) {
+    const tgt = e.currentTarget;
+    let flag = false;
+
+    switch (e.key) {
+      case 'ArrowLeft':
+        this.moveFocusToPreviousTab(tgt);
+        flag = true;
+        break;
+
+      case 'ArrowRight':
```

Comment:
i thnk this navigation has a minor flaw, when in mobile you reach the "collapsed" tabs you would maybe expect to be able to browse the items also using arrowDown and arrowUp, no..? At least this is what i instinctively tried to do when i first tested this.

Reply:
I was first trying doing this, but it's more simple now the dropdown opens automatically if you try to go to the next tabs (collapse). Otherwise, you need to tab open the dropdown navigate with up and down arrows and tab again to come back to previous tabs. this was agreed with COMM.

Reply:
i was only sugggesting to "add" the possibility of browsing the dropdown also with up and down arrows, nothing wrong with the fact that it opens and close using left and right arrows